region/lowernorfair/east/Plowerhouse Room.json

```diff
@@ -485,29 +665,32 @@
       "link": [2, 3],
       "name": "Power Beam Kill",
       "requires": [
-        "h_canNavigateHeatRooms",
+        {"or": [
+          "canBePatient",
+          "canDodgeWhileShooting"
+        ]},
         {"heatFrames": 2400}
```

Comment:
Not sure on the patience here. With these heat frames it is only 40 seconds. Unless the frames need to be extended as well. I'm assuming this is supposed to be modeling aiming while it's swooping? (Not that it matters much, but in Map Rando, this would already require 2400 Energy on Basic and 1500 on Medium before `canDodgeWhileShooting` was added.)

Reply:
The intent of both techs is that shooting for 40s is tedious.  I think you should be expected to have at least another beam on low difficulties.  Its hard to predict how someone would try killing the Holtzes without knowing it is safe by the door, but you figure that out whenever the left one starts moving.  There is an aiming component, so it could take a little more tan 40 seconds.

40s is a good estimate, you can speed it up a little bit by stacking the holtzes.  But if you know how to do that you would have a h_heatResistant to farm or could get them stuck.

Right now all the kill strats have a heat resistant requirement, mostly because its needed to use the farm points, but maybe also because crossing the room with nothing was in medium.